manifests/generated/node-maintenance-operator.vVERSION.clusterserviceversion.yaml

```diff
@@ -2,8 +2,10 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[{"apiVersion":"nodemaintenance.kubevirt.io/v1beta1","kind":"NodeMaintenance","metadata":{"name":"nodemaintenance-xyz"},"spec":{"nodeName":"node02","reason":"Test
-      node maintenance"}}]'
+    alm-examples: "[\n  {\n    \"apiVersion\": \"kubevirt.io/v1alpha1\",\n    \"kind\"\
+      : \"NodeMaintenance\",\n    \"metadata\": {\n      \"name\": \"nodemaintenance-xyz\"\
+      \n    },\n    \"spec\": {\n      \"nodeName\": \"node02\",\n      \"reason\"\
+      : \"Test node maintenance\"\n    }\n  }\n]"
```

Comment:
The tool did this??

Reply:
You are right, we have merged in a change that changed the apiversion field. Now that commit had issues and broke CSV generation, PR #75 tackles these issues, so I incorporated the changes of #75 into this PR as well